API/Hardware_API.md

```diff
@@ -66,18 +66,58 @@ By default, all of the pins are pulled high if not specifically set.
 
 ### Digital pins
 
-A digital pin (any pin other than 3.3V and GND on Tessel 2) is either high (on/3.3V) or low (off/0V). On both of ports A and B, pins 0 and 1 are pulled high to 3.3V by default.
+A digital pin (any pin other than 3.3V and GND on Tessel 2) is either high (on/3.3V) or low (off/0V). On both of ports A and B, all pins are pulled high to 3.3V by default.
 
-Here is an example usage of a digital pin on Tessel:
+**pin.write:**
+
+Set the digital value of a pin.
+
+```js
+pin.write(number, callback(error, buffer));
+```
+
+option    | type     | description                   | required
+----------|----------|-------------------------------|---------
+number    | Number   | 0 for "low", 1 for "high"     | yes
+callback  | Function | Called when write is complete | yes
+
+Example:
 
 ```js
 var tessel = require('tessel'); // import tessel
 var pin = tessel.port.A.pin[2]; // select pin 2 on port A
-pin.output(1);  // turn pin high (on)
-pin.read(function(error, value) {
-  // print the pin value to the console
-  console.log(value);
-  pin.output(0);  // turn pin low (off)
+pin.write(1, (error, buffer) => {
+  if (error) {
+    throw error;
+  }
+
+  console.log(buffer.toString()); // log the value
```

Comment:
Is it necessary to `toString()` this? When newcomers copy/paste our example code, they often have trouble because they are trying to modify from the example and don't know what all the functions do– accelerometer's `toFixed` is the most common example of this issue. For this reason, it might be better to take out any unnecessary complexities.